custom_components/tuya_local/devices/aubess_rainpoint_irrigation_system.yaml

```diff
@@ -0,0 +1,140 @@
+name: Aubess Irrigation System 
```

Comment:
Home Assistant naming convention is to only capitalise the first word in a name, unless it is a trademark or abbreviation that needs to be capitalised.  I have also started to prefer not using brand/model names in the name, as there are often other brands of the same device sold in other markets or by different retailers in the same market. So just "Irrigation system" may be better here, and the model and manufacturer will be picked up later  from the products entry once discovery is implemented properly.